packages/tools/client-debugger/client-debugger-chrome-extension/src/react-components/AudienceView.tsx

```diff
@@ -86,15 +91,30 @@ export function AudienceView(props: AudienceViewProps): React.ReactElement {
 		};
 	}, [containerId, setAudienceState, setAudienceHistory]);
 
+	console.log("audienceState:", audienceState);
+	console.log("--------------------------");
+	console.log("audienceHistory:", audienceHistory);
+
 	return (
 		<Stack>
 			<StackItem>
 				<h3>Audience Data</h3>
 			</StackItem>
 			<StackItem>
-				<div>TODO</div>
-				<div> {audienceState} </div>
-				<div> {setAudienceHistory} </div>
+				<div>
```

Comment:
With `_AudienceView` exported from the client-debugger-view package, this `return` should be identical to the one over there, just returning that view.

Reply:
Now the `chrome-extension/AudienceView` returns `<_AudienceView/>` by sending the following props:
- `clientId`
- `audienceClientMetaData`
- `onRenderAudienceMember`
- `audienceHistory`

`client-debugger-view/AudienceView` initializes `myClientConnection` inside the `_AudienceView` which is derived from `audienceClientMetaData`.